src/wallet/wallet.h

```diff
@@ -875,8 +877,11 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     /* Returns the wallets help message */
     static std::string GetWalletHelpString(bool showDebug);
 
-    /* initializes the wallet, returns a new CWallet instance or a null pointer in case of an error */
-    static CWallet* InitLoadWallet(bool fDisableWallet, const std::string& strWalletFile, std::string& warningString, std::string& errorString);
+    /* Initializes the wallet, returns a new CWallet instance or a null pointer in case of an error */
+    static bool InitLoadWallet();
```

Comment:
> Why static instead of calling it from the pwalletMain global (which could be moved to wallet)?
> Or are we turning the global into a singleton?

Right. This is the direction where I'm heading to. But I work in very tiny steps (as you can see). For now it just moves "static code" to a static function in CWallet. Later it should be within the `pwalletMain` context.